tests/ci-helper.test.ts

```diff
@@ -35,38 +35,39 @@ async function getSMTPInfo():
 
 class TestCIHelper extends CIHelper {
     public ghGlue: GitHubGlue;      // not readonly reference
-    public addPRComment: any;
-    public updatePR: any;
+    public addPRComment = jest.fn();
+    public updatePR = jest.fn();
 
     public constructor(workDir?: string, debug = false, gggDir = ".") {
         super(workDir, debug, gggDir);
         this.testing = true;
         this.ghGlue = this.github;
-        this.addPRComment = jest.fn();
-        this.ghGlue.addPRComment = this.addPRComment;
-        this.updatePR = jest.fn();
-        this.ghGlue.updatePR = this.updatePR;
+        Object.assign(this.ghGlue, this);
```

Comment:
Is this risky?  `this` is unrelated to `GitHubGlue` and could override something unexpectedly.  Yes, not a problem now but a possible future problem.

Reply:
My thinking on this is: even if we copy unrelated attributes, they are simply not used in `this.ghGlue`, so they don't hurt.

Reply:
`workDir` is a member common to both objects (GitHubGlue and CIHelper).  Yes, it would cause no harm in this instance.  More comments to come.

Reply:
More along the lines of what I would have done originally if I had worked harder (thank you for that now), see [this alternative update](https://github.com/webstech/gitgitgadget/commit/e1d845ca52c6ec808cd250cd4c35e6b732e64256).  

What do you think?

Reply:
Yes, I think that's a good direction to take.

Reply:
Are you adding to this PR?  If not, I will approve and put the changes in a new PR.

Reply:
> Are you adding to this PR? If not, I will approve and put the changes in a new PR.

Sorry, I had not seen the question in the alternative.

Reply:
I will add it to this PR.